frontend/src/components/GrantPermissionDialog.vue

```diff
@@ -96,11 +96,11 @@ async function grantAccess() {
   }
 }
 
-async function giveDevicesAccess(devices: DeviceDto[]) {
-  for (const device of devices) {
-    const publicKey = base64url.parse(device.publicKey);
-    const jwe = await props.vaultKeys.encryptForDevice(publicKey);
-    await backend.vaults.grantAccess(props.vault.id, device.id, jwe, props.vaultKeys);
+async function giveUsersAccess(users: UserDto[]) {
+  for (const user of users) {
+    const publicKey = base64.parse(user.publicKey!); // TODO: can we be sure the user has a public key?
```

Comment:
Does `User.requiringAccessGrant` only return users with a public key?

Reply:
fixed in bf4a625